### DISCUSSION TITLE
Screenshot without home indicator

### DISCUSSION THREAD

**Discussion by gran33** [fastlane/fastlane#17519] [Ideas]
Hey,
I try to take screenshots with `Snapshot` but I want to hide the home indicator.
Any option to do that?

Thanks in advance!

**Comment by rogerluan** [comment]
Hey @gran33 ! 👋 

I'm unaware if this is possible via simulator command line tools, but you can achieve this in the code if you wish: https://stackoverflow.com/questions/46692121/iphone-x-hide-home-indicator-on-view-controller

Maybe you could add a runtime check that will display the home indicator only when not running the app for the screenshots 😊 

Good luck!
